server/src/test/java/org/elasticsearch/search/query/SearchQueryIT.java

```diff
@@ -1757,5 +1757,5 @@ public void testFieldAliasesForMetaFields() throws Exception {
 
         DocumentField field = hit.getFields().get("id-alias");
         assertThat(field.getValue().toString(), equalTo("1"));
-   }
+    }
```

Comment:
maybe undo changes to this file since they are unrelated

Reply:
👍